### ISSUE TITLE
Marker clusterer

### ISSUE THREAD

**Issue by digitalsadhu** [mediasuitenz/mappy#38]
We need to be able to define marker clustering via configuration.

How it could work:

Define a layer config postProcess block
```js
[{
  //....
  postProcess: [ //should this be called 'postProcess'? Maybe 'transforms' or 'plugins'?
  {
    type: cluster,
    levels: {
      on: 5,
      off: 10
    }
    icon: {
      //the usual icon definition
    }
  }],
  //...
}]
```

We are likely to have other things much like this to add later eg. heatmap
So we should make this definition as reusable as possible etc.

@markstuart (and anyone else) thoughts?

**Comment by benmanu** [comment]
Hey @digitalsadhu we are looking at implementing clustering and was wondering if there has been any progress on your guys end with implementation?

**Comment by digitalsadhu** [comment]
Yup I think it's nearly done. Will check where progress is at and get back

**Comment by digitalsadhu** [comment]
@benmanu So @Resonance1584 is off the grid climbing in Laos.

He doesn't seem to have committed his work on clustering and we may not see him at all until the start of Feb.

So that only thing I can suggest is that either this isn't too time sensitive for you and maybe wait and see if @Resonance1584 appears or if it is you may have to implement yourselves. Sorry about that.

**Comment by benmanu** [comment]
Thanks for the update @digitalsadhu, will look at a plan of attack on our end.

**Comment by digitalsadhu** [comment]
Cool, will let you know if @Resonance1584 shows up early.

PS. We have added a gitter chatroom for the project and added you guys. Feel free to jump in there if you want to discuss anything. Theres a badge on README.md that will get you there

**Comment by digitalsadhu** [comment]
Clustering is now implemented in mappy via #75 #77 #79
